recipes/genomekit/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set version = "4.1.7" %}
+
+package:
+  name: genomekit
+  version: {{ version }}
+
+source:
+  url: https://github.com/deepgenomics/GenomeKit/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 9f663c750e53585411acd477a5f6954131cb402b787e85beef4753f23ac67b93
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  # tensorflow TPU whls from google only support 3.8
+  # TODO tensorflow TPU now also supports 3.10
+  skip: true  # [py<38 or py>38]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+
+  host:
+    - appdirs
+    - fmt
+    - numpy >=1.18.0,<1.19.0 # oldest py3.8 build for forward compatibility
+    - typing-extensions
+    - zlib
+    - python
+    - pip
+
+  run:
+    - appdirs
+    - crc32c
+    - google-cloud-storage
+    - {{ pin_compatible('numpy') }}
+    - typing-extensions
+    - importlib-metadata
+    - tqdm
+    - python
+
+test:
+  imports:
+    - genome_kit
+  requires:
+    - pytest-cov
+    - pip
+  source_files:
+    - tests
+  commands:
+    # can't require bioconda packages in the requires section
+    - "conda install -c bioconda bcftools samtools>=1.15 tabix twobitreader"  # [not win]
```

Comment:
No. If this depends on bioconda packages it either needs to go into bioconda or all of the dependencies must be copied over to conda-forge.

Reply:
Rather than doing that, I'm going to skip the few tests that use that dependency. I'd still like to run them on our own CI, though. Can you suggest an env var to check for?